src/modules/commands/duel.js

```diff
@@ -0,0 +1,36 @@
+const isEmpty = require('lodash/isEmpty');
+const command = require('../command.filter');
+
+const duelling = async function (request, context) {
+    const { i18n, send, getUser } = context;
+    const { user, args: userId } = request;
+
+    const user1 = user;
+    const user2 = await getUser(userId);
+
+    if (isEmpty(user2)) {
+        send(i18n('duel.userisnotdefined', { id: user1 }));
+
+        return request;
+    }
+
+    if (user2 === user1) {
+        send(i18n('duel.cantduelauthor', { id: user1 }));
+
+        return request;
+    }
+
+    const users = [];
+    await users.push(user1);
+    await users.push(user2);
+
+    const winner = users[Math.floor(Math.random * users.length)];
```

Comment:
ух, не люблю такие грязные конструкции) используй sample is lodash
```js
import sample from 'lodash/sample';
```
```suggestion
    const winner =  sample(users);
```

Reply:
Тебе будет больно смотреть мой PR с игрулькой XD

Reply:
кмон) больно смотреть, когда код красивый и не работает... и вроде все ок, а не работает) а когда 3-4 операции в одной строке, это просто мутно

Reply:
Вот это альтернатива :0 теперь буду знать